server/controllers/notifications.js

```diff
@@ -1,14 +1,42 @@
 const __ = require('config').universalPath
 const _ = __.require('builders', 'utils')
 const error_ = __.require('lib', 'error/error')
+const sanitize = __.require('lib', 'sanitize/sanitize')
 const responses_ = __.require('lib', 'responses')
 const notifs_ = __.require('lib', 'notifications')
 
+const sanitization = {
+  limit: { optional: true, default: 10 },
+  offset: { optional: true },
+}
+
+const Paginate = page => notifications => {
```

Comment:
```suggestion
const paginate = params => notifications => {
```
Thanks for posting this because we can now update the user contribution guidelines (after https://github.com/inventaire/jingo/issues/6) with this kind of naming conventions that are not really documented, 
The suggestion above is welcome but not mandatory, it may be merged like this in a few days if you dont feel like modifying it.